### ISSUE TITLE
HIV FHIR bundle

### ISSUE THREAD

**Issue by lg-king** [CDCgov/prime-simplereport#6582]
**blocked**: waiting for feedback from RS on where to include gender of sexual partners in FHIR bundle

### Description
Create HIV FHIR bundle

HIV data fields - https://docs.google.com/spreadsheets/d/1BeotxqF138gSUkNZ-LsNVzYVfi88v6XxHTAQImHWXKU/edit?usp=sharing

Slack thread with docs: https://skylight-hq.slack.com/archives/C02HKNHGCP6/p1688656583617709

### Acceptance criteria
- Create/modify FHIR bundle as needed based on new requirements - AOE (not LOINCs), gender identity

### Notes
Coordinate with HIV AOE ticket
Status 11/1: Working on AOE, coordinating with RS/LAC on where these should go. Otherwise bundle is the same, includes new fields. Dependent on RS. 

Work remaining:
- add gender of sexual partners to fhir bundle
- ~are we adding comment in HIV AOE and in fhir bundle~

**Comment by mehansen** [comment]
Update: remaining work is to add gender of sexual partners to the FHIR bundle - we're still waiting on feedback from RS on where to put this so this is *blocked*

**Comment by mehansen** [comment]
Closing this as complete and making a separate issue specifically for adding gender of sexual partners to our FHIR bundle because I don't want to overwrite the info and history in this ticket
